release/OpenWrt/files/hetzner_ddns

```diff
@@ -0,0 +1,23 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+USE_PROCD=1
+
+start_service() {
+         procd_open_instance [hetzner_ddns]
+         procd_set_param command /usr/bin/hetzner_ddns -d /etc/config/hetzner_ddns.conf
+         
+         # respawn automatically if something died, be careful if you have an alternative process supervisor
+         # if process exits sooner than respawn_threshold, it is considered crashed and after 5 retries the service is stopped
+         # if process finishes later than respawn_threshold, it is restarted unconditionally, regardless of error code
+         # notice that this is literal respawning of the process, not in a respawn-on-failure sense
+         procd_set_param respawn ${respawn_threshold:-3600} ${respawn_timeout:-5} ${respawn_retry:-5}
+
+         procd_set_param limits core="unlimited"
```

Comment:
If you are applying resource limits, I suggest to go all the way: one core shall be plenty.

Reply:
Ah, I forgot about that. It is changed to "0" with [bea11d1](https://github.com/filiparag/hetzner_ddns/pull/18/commits/bea11d13da88fc315202d75d320456a2b4a1dcd4).

This parameter is for core dumps and for debugging creating unlimited core dump sizes seemed ok. But now as you mentioned it I would go with 0 as there is not many disk space on smaller routers. 

Or would you like to omit the value completely? That would be also an option.